Snippets/Rabbit/Rabbit_4/Usage.cs

```diff
@@ -140,4 +141,14 @@ void PrefetchCount(EndpointConfiguration endpointConfiguration)
         #endregion
     }
 
+    void SetClientCertificates(EndpointConfiguration endpointConfiguration)
+    {
+        #region rabbitmq-config-client-certificates
+
+        var transport = endpointConfiguration.UseTransport<RabbitMQTransport>();
+        transport.SetClientCertificates(new X509CertificateCollection());
```

Comment:
Discussed this with @SimonCropp: perhaps we need to show something other than the setting of an empty collection.